framework/src/main/java/org/checkerframework/framework/type/TypeFromMemberVisitor.java

```diff
@@ -72,10 +75,30 @@ public AnnotatedTypeMirror visitMethod(MethodTree node, AnnotatedTypeFactory f)
         result.setElement(elt);
 
         ElementAnnotationApplier.apply(result, elt, f);
+        new TypeVarAnnotator().visit(result, f);
```

Comment:
Instead of creating a new `TypeVarAnnotator` every time a method is visited, couldn't you create a single instance and then call `reset()` before every usage?

Reply:
Yes, I'll do that once the above discussion is settled.